install/kubernetes/addons/prometheus.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: ConfigMap
```

Comment:
all those now generated files should have the header/comment indicating that they are or people are going to manually edit them (or we should delete them from source control)